_vector-search/ai-search/hybrid-search/inner-hits.md

```diff
@@ -0,0 +1,927 @@
+                  "_source": {
+                    "name": "John Alder",
+                    "age": 35
+                  }
+                }
+              ]
+            }
+          }
+        }
+      },
+      {
+        "_index": "my-nlp-index",
+        "_id": "2",
+        "_score": 5.0E-4,
+        "_source": {
+          "user": [
+            {
+              "name": "John Wick",
+              "age": 46
+            },
+            {
+              "name": "John Snow",
+              "age": 40
+            },
+            {
+              "name": "Sansa Stark",
+              "age": 22
+            },
+            {
+              "name": "Arya Stark",
+              "age": 20
+            }
+          ],
+          "location": [
+            {
+              "city": "Tromso",
+              "state": "Norway"
+            },
+            {
+              "city": "Los Angeles",
+              "state": "California"
+            },
+            {
+              "city": "London",
+              "state": "UK"
+            }
+          ]
+        },
+        "inner_hits": {
+          "location": {
+            "hits": {
+              "total": {
+                "value": 0,
+                "relation": "eq"
+              },
+              "max_score": null,
+              "hits": [
+
+              ]
+            }
+          },
+          "user": {
+            "hits": {
+              "total": {
+                "value": 2,
+                "relation": "eq"
+              },
+              "max_score": 0.31506687,
+              "hits": [
+                {
+                  "_index": "my-nlp-index",
+                  "_id": "2",
+                  "_nested": {
+                    "field": "user",
+                    "offset": 0
+                  },
+                  "_score": 0.31506687,
+                  "_source": {
+                    "name": "John Wick",
+                    "age": 46
+                  }
+                },
+                {
+                  "_index": "my-nlp-index",
+                  "_id": "2",
+                  "_nested": {
+                    "field": "user",
+                    "offset": 1
+                  },
+                  "_score": 0.31506687,
+                  "_source": {
+                    "name": "John Snow",
+                    "age": 40
+                  }
+                }
+              ]
+            }
+          }
+        }
+      }
+    ]
+  }
+}
+```
+
+## Understanding the relevancy between inner_hits and parent documents by passing `explain` flag
+User can pass `explain` flag in the search request to understand the search response formation. To know more about using explain api with hybrid query, read [Hybrid score explanation processor]({{site.url}}{{site.baseurl}}/search-plugins/search-pipelines/explanation-processor/) and [Hybrid search explain]({{site.url}}{{site.baseurl}}/_vector-search/ai-search/hybrid-search/explain/).
```

Comment:
we need to add part where user needs to update the search pipeline by adding the hybrid_score_explanation response processor. If they do everything as written in this PR then explain will not work because the response processor is missing from the initial pipeline definition,  and we're re-using same pipeline for examples with explain.